### ISSUE TITLE
[BUG] Bootloops when no TV is on (HDMI disconnected)

### ISSUE THREAD

**Issue by Enigma0** [ValveSoftware/SteamOS#550]
#### Your system information

* Steam client version: latest
* SteamOS version: latest (2.87) and earlier - happened as early as 2.7x also
* Opted into Steam client beta?: Yes
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
If I reboot or power on the Steam Machine without the TV already on, the system bootloops endlessly.

#### Steps for reproducing this issue:

1. Turn off Steam machine
2. Turn off TV
3. Turn on Steam machine
4. Wait up to several minutes for bootloop to occur

**Comment by sasann** [comment]
Does steamos boots successfully  after you turn off the machine, turn on tv then turn the machine on again?

**Comment by Enigma0** [comment]
@sasann Yup. It's related to an EDID issue (display doesn't exist when TV is off).
